app/assets/stylesheets/cards.scss

```diff
@@ -0,0 +1,14 @@
+// Place all the styles related to the cards_controller controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.card {
+  height: 150px;
```

Comment:
Properties should be ordered background-color, border, color, cursor, font-size, height, transition, width